curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5f0d4ab1b435f13ab6550052.md

```diff
@@ -50,13 +50,13 @@ assert(foundElems.length === 0);
 Both `fieldset` elements should be above the text field. They are out of order.
 
 ```js
-const formChildren = $('form')[0].children;
-assert(
-  formChildren[0].nodeName === 'FIELDSET' &&
-    formChildren[1].nodeName === 'FIELDSET' &&
-    formChildren[2] &&
-    formChildren[2].nodeName === 'INPUT' &&
-    formChildren[2].getAttribute('type') === 'text'
+const formChildren = document.querySelector('form')?.children;
+assert.isTrue(
+  formChildren?.[0]?.nodeName === 'FIELDSET' &&
+  formChildren?.[1]?.nodeName === 'FIELDSET' &&
+  formChildren[2] &&
```

Comment:
```suggestion
```
This can be dropped now, `formChildren?.[2]?.(...)` will be handling cases when there actually isn't `formChildren[2]`.